packages/react/tabs/src/Tabs.tsx

```diff
@@ -79,14 +73,13 @@ const Tabs = React.forwardRef((props, forwardedRef) => {
   });
 
   const ctx: TabsContextValue = React.useMemo(
-    () => ({ tabsId, value, setValue, orientation, activationMode }),
-    [activationMode, orientation, value, setValue, tabsId]
+    () => ({ baseId, value, setValue, orientation, activationMode }),
+    [baseId, value, setValue, orientation, activationMode]
   );
 
   return (
     <TabsContext.Provider value={ctx}>
       <Primitive
-        id={tabsId}
```

Comment:
I'm guessing this one was redundant?

Reply:
yeah it's not really needed at all for accessibility.